fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/RegistryAttributeRegistry.java

```diff
@@ -0,0 +1,20 @@
+package net.fabricmc.fabric.api.event.registry;
+
+import net.fabricmc.fabric.impl.registry.sync.RegistryAttributeRegistryImpl;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+//TODO better class name here lol
+public interface RegistryAttributeRegistry {
+
+	RegistryAttributeRegistry INSTANCE = RegistryAttributeRegistryImpl.INSTANCE;
+
+	void registerAttributes(RegistryAttribute attribute, Registry<?>... registries);
```

Comment:
I'd prefer one
```java
void registerAttributes(Registry<?> registry, RegistryAttribute attribute);
void registerAttributes(Registry<?> registry, RegistryAttribute... attributes);
```
instead

Also consider turning these attributes into like keys, like `RegistryAttribute<T>` where `T` may carry custom values for properties

Reply:
Possibly, I want to rethink this this part of the API.